### PR TITLE
Document how to use FIXME and TODO

### PR DESCRIPTION
**Pull request by mavam**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -69,6 +69,26 @@ guide](https://github.com/agis/git-style-guide).
...
+```cpp
+// FIXME: this currently fails on FreeBSD. 
+// FIXME: this algorithms is broken for i < 0.
+```
+
+A typical `TODO` could be:
+
+```python
+# TODO: refactor this code to separate mechanism from policy
+# TODO: add another argument to process user-defined tags
```
Comment (dominiklohmann) [inline contributing.md]: These should be full sentences: Capitalized at the start and a dot at the end.

**Review by dominiklohmann — CHANGES_REQUESTED** [review]

File: contributing.md (hunk 1 above)
Comment (mavam) [reply contributing.md]: I agree with your regarding the missing punctuation. However, the capitalization after a colon is less established. I have come across both styles, although predominantly seen lowercase.

**Review by 0snap — APPROVED** [review]

File: contributing.md (hunk 1 above)
Comment (0snap) [reply contributing.md]: I can agree to both, caps or not. The dot at the end, however, is more important, as it keeps the sentence short and makes it a statement.

Comment (mavam) [reply contributing.md]: Yeah, maybe we can simply leave the capitalization open, since it's gramatically not a clear black-vs-white issue?

Comment (dominiklohmann) [reply contributing.md]: I agree. Merging this.

**Pull request merged**